intel_owl/celery.py

```diff
@@ -69,4 +69,10 @@
         "schedule": crontab(minute=0, hour=0),
         "options": {"queue": "default"},
     },
+    # execute 1 time a day
```

Comment:
We should not leverage celery crons to do this. A lot of IntelOwl users could never use FireHol lists. This could be an incredible amount of useless computation and API abuse.

Please check how [stratosphere analyzer](https://github.com/intelowlproject/IntelOwl/blob/master/api_app/script_analyzers/observable_analyzers/stratosphere.py) was implemented and change the analyzer accordingly

Reply:
Well actually it was my idea to use celery cron but yes I did not think about the drawback you mention. 

@subhashissuara We can dynamically download the IP list only when the user requests the analysis. So your `updater` method would just be a `download_ipset` method which would get called at the beginning of `run`.